### ISSUE TITLE
[iceprod2] jobs and the server ssl cert (Trac #1498)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1490]
Each job needs the public copy of the server ssl cert to verify it.  Attach that somewhere in the submission process, and actually use it when talking to the server.

Note that this isn't an issue for servers with real signed certificates, but a problem for self-signed or strange CAs.  Since the default is to self-sign, this will be common.

Migrated from https://code.icecube.wisc.edu/ticket/1498
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Each job needs the public copy of the server ssl cert to verify it.  Attach that somewhere in the submission process, and actually use it when talking to the server.\n\nNote that this isn't an issue for servers with real signed certificates, but a problem for self-signed or strange CAs.  Since the default is to self-sign, this will be common.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] jobs and the server ssl cert", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-01-04T19:52:59", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160105T21:56:19: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 140727:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140727"
> use ssl in the core. always send a cfg with a job, even if it's a pilot, to set the options. get the server to submit proper x509 and ca certs. closes #1490
> ```


* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
